src/i_video.c

```diff
@@ -1417,27 +1416,30 @@ static void UpdateViewport(void)
 
     if (fabs(want_aspect - real_aspect) < 0.0001)
     {
+        scale = (float)w / video.width;
```

Comment:
The scales should differ here, we are stretching to cover the window

Reply:
Okay, how about this: https://github.com/fabiangreffrath/woof/pull/1847/commits/6aabb3f181044583f76a675d7d9398f5bcc4fa0c

Interestingly, all of the variants work either way 😄